nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -58,6 +58,19 @@ in {
         '';
       };
 
+      sessionManager = {
+        enable = mkEnableOption "pipewire session manager";
+
+        package = mkOption {
```

Comment:
This creates two different ways to set the session manager. Given that pipewire itself is not configured in any way by this, I think it might be cleanest to just install each session manager individually in its own module. This kind of de-duplication feels forced to me anyway.

Reply:
The idea was to have a working session manager by default and not require looking where it needs to be disabled when switching to a different one. Granted, it is a bit redundant if media-session is going to be removed as soon as wireplumber is usable.